src/backend/joanie/payment/models.py

```diff
@@ -390,6 +390,7 @@ class CreditCard(BaseModel):
         on_delete=models.CASCADE,
     )
     is_main = models.BooleanField(_("main"), default=False)
+    payment_provider = models.CharField(_("payment provider"), max_length=50, null=True, blank=True)
```

Comment:
I don't think this field should be nullable.

Reply:
In a blue/green strategy yes it should. Otherwise you can create new row in the table while you didn't switch. In django 5.0, using [db_default](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.Field.db_default) should help us, but for now we can't.

In a blue/green strategy, this field should be change to be required in a future release.